docs/ui-review/148/README.md

```
@@ -0,0 +1,69 @@
+# #148 終局後に盤面が縮む — 画面確認（五目並べ・マインスイーパー・麻雀ソリティア・オセロ・2048）
+
+#139（将棋）で入れた手当ての横展開。盤（またはカードグリッド）は**塗り色の外接矩形を
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)